Misc/NEWS.d/next/Core and Builtins/2022-04-20-14-43-37.gh-issue-91632.cvUhsZ.rst

```diff
@@ -0,0 +1 @@
+Fix :class:`generic_alias_iterator` to be finalized at exit.
```

Comment:
I don't understand this NEWS entry. I'm not sure that it's worth it to document this minor fix, since only a minority of users embed Python in an application and continue to run the application once Python is finalized. Usually, the application stops when Python exits.

If you want to document it, I suggest something like:

```suggestion
Fix a minor memory leak at exit: release the memory of the :class:`generic_alias_iterator` type. Patch by Dong-hee Na.
```